docs/updating/from_4.0/to_4.1.md

```diff
@@ -0,0 +1,121 @@
+---
+latest_tag: '4.1.1'
+---
+
+# Update from v4.0.x to v4.1
+
+This update procedure applies if you are using v4.0.latest.
+
+Go through the following steps to update to v4.1.
+
+## Update the app to v4.1
+
+First, run:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer require ibexa/content:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer require ibexa/experience:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer require ibexa/commerce:[[= latest_tag =]] --with-all-dependencies --no-scripts
+    ```
+
+Continue with updating the app:
+
+=== "[[= product_name_content =]]"
+
+    ``` bash
+    composer recipes:install ibexa/content --force -v
+    ```
+
+=== "[[= product_name_exp =]]"
+
+    ``` bash
+    composer recipes:install ibexa/experience --force -v
+    ```
+
+=== "[[= product_name_com =]]"
+
+    ``` bash
+    composer recipes:install ibexa/commerce --force -v
+    ```
+
+The `recipes:install` command installs new YAML configuration files. Look through the old YAML files and move your custom configuration to the relevant new files.
+
+## Update the database
+
+[[% include 'snippets/update/db/db_backup_warning.md' %]]
+
+Apply the following database update script:
+
+### Ibexa DXP
+
+=== "MySQL"
+    ``` bash
+    mysql -u <username> -p <password> <database_name> < vendor/ibexa/installer/upgrade/db/mysql/ibexa-4.0.0-to-4.1.0.sql
+    ```
+
+=== "PostgreSQL"
+
+    ``` bash
+    psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.0.0-to-4.1.0.sql
+    ```
```

Comment:
Instead of calling database specific clients with their secrets (passwords), one can use
```
php bin/console dbal:run-sql "$(< vendor/ibexa/installer/upgrade/db/postgresql/ibexa-4.0.0-to-4.1.0.sql)"
```
and similar.

Do note that this needs to be tested, because it requires a working compiled kernel (cache).

Reply:
We've been advising customers to use the command in its original form ever since v2.5, so let's stick to that. Unless you have time to test the new variant?